src/component/ResponsiveContainer.tsx

```diff
@@ -162,19 +162,21 @@ export const ResponsiveContainer = forwardRef<HTMLDivElement, Props>(
       );
 
       return React.Children.map(children, child => {
-        return cloneElement(child, {
-          width: calculatedWidth,
-          height: calculatedHeight,
-          // calculate the actual size and override it.
-          style: {
-            height: '100%',
-            width: '100%',
-            maxHeight: calculatedHeight,
-            maxWidth: calculatedWidth,
-            // keep components style
-            ...child.props.style,
-          },
-        });
+        if (React.isValidElement<any>(child)) {
```

Comment:
in React 19 this check requires that your `react-is` version exactly match your `react/react-dom` versions. Not everyones always does so this will cause issues